### PR TITLE
polarssl: fix sha256

### LINKED ISSUE #36844: PolarSSL bottles have wandered off
```
brew install -v polarssl
==> Downloading https://downloads.sf.net/project/machomebrew/Bottles/polarssl-1.3.10.yosemite.bottle.tar.gz
/usr/bin/curl -fLA Homebrew 0.9.5 (Ruby 2.0.0-481; Mac OS X 10.10.3) https://downloads.sf.net/project/machomebrew/Bottles/polarssl-1.3.10.yosemite.bottle.tar.gz -C 0 -o /Library/Caches/Homebrew/polarssl-1.3.10.yosemite.bottle.tar.gz.incomplete
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:--  0:00:03 --:--:--     0curl: (22) The requested URL returned error: 404 Not Found
Error: Failed to download resource "polarssl"
Download failed: https://downloads.sf.net/project/machomebrew/Bottles/polarssl-1.3.10.yosemite.bottle.tar.gz
```

```
~> lwget https://downloads.sf.net/project/machomebrew/Bottles/polarssl-1.3.10.mavericks.bottle.tar.gz
--2015-02-16 04:36:32--  https://downloads.sf.net/project/machomebrew/Bottles/polarssl-1.3.10.mavericks.bottle.tar.gz
Resolving downloads.sf.net... 216.34.181.129
Connecting to downloads.sf.net|216.34.181.129|:443... connected.
HTTP request sent, awaiting response... 404 Not Found
2015-02-16 04:36:36 ERROR 404: Not Found.
```

### PR DESCRIPTION
**Pull request by DomT4**
Closes #36844

References:
* https://github.com/polarssl/polarssl/issues/173
* https://polarssl.org/tech-updates/releases/mbedtls-1.3.10-released

### CODE REVIEW

File: Library/Formula/polarssl.rb [hunk 1]
```diff
@@ -29,9 +29,9 @@ def install
     system "make"
     system "make", "install"
     # Why does PolarSSL ship with GNU's Hello included? Let's remove that.
-    rm "#{bin}/hello"
+    rm_f "#{bin}/hello"
     # Remove the pointless example application that hooks into system OpenSSL
-    rm "#{bin}/o_p_test"
+    rm_f "#{bin}/o_p_test"
```
Comment (jacknagel) [inline Library/Formula/polarssl.rb]: Why this change (just curious)?

Comment (DomT4) [inline Library/Formula/polarssl.rb]: Mostly paranoia on my part. Upstream are making some fairly big changes, and if either of these elements gets removed/changed in the ` head ` rm will error out, whereas ` rm_f ` should plow on regardless, I believe.
